pennylane/ops/op_math/controlled_ops.py

```diff
@@ -531,7 +531,24 @@ def _cy(wires: WiresLike, **__):
     qml.S(wires=wires[0])
 
 
-add_decomps(CY, _cy)
+def _cy_to_ppr_resource():
+    return {
+        resource_rep(qml.PauliRot, pauli_word="IY"): 1,
+        resource_rep(qml.PauliRot, pauli_word="ZI"): 1,
+        resource_rep(qml.PauliRot, pauli_word="ZY"): 1,
+        qml.GlobalPhase: 1,
+    }
+
+
+@register_resources(_cy_to_ppr_resource)
+def _cy_to_ppr(wires: WiresLike, **_):
+    qml.PauliRot(-np.pi / 2, "IY", wires=wires)
+    qml.PauliRot(-np.pi / 2, "ZI", wires=wires)
+    qml.PauliRot(np.pi / 2, "ZY", wires=wires)
+    qml.GlobalPhase(np.pi / 4)
+
+
+add_decomps(CY, _cy, _cy_to_ppr)
```

Comment:
According to [this page](https://pennylane.ai/compilation/pauli-product-rotations/static-gates) the CY decomposition should perform `pi/4` rotations. Just trying to make sure I don't miss anything, but I see that `PauliRot`'s [generator](https://github.com/PennyLaneAI/pennylane/blob/fbb1b8e0275c46944eb5ae2e67b441ada83228a1/pennylane/ops/qubit/parametric_ops_multi_qubit.py#L475) has a coefficient of `-0.5`. I'm assuming that is why you're using `pi/2` rotations instead of `pi/4`?